test/growth.jl

```diff
@@ -8,7 +8,7 @@ using ADRIA
     max_cover = rand(20)
 
     for i in axes(Y, 1)
-        ADRIA.proportional_adjustment!(Y[i, :, :], tmp, max_cover)
+        ADRIA.proportional_adjustment!(Y[i, :, :], max_cover)
```

Comment:
I don't know what happened here, but the original was correct. It looks like the changes were left out during an incorrect merge/rebase.